tool/mediapipe_ci_script_stable.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -e
+# set -e
```

Comment:
++

Reply:
This change is temporary to increase debugging visibility into what's happening on GitHub Actions machines